src/Model/ValueObject/Web/Domain.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Model\ValueObject\Web;
+
+use CultuurNet\UDB3\Model\ValueObject\String\Behaviour\IsString;
+
+abstract class Domain
```

Comment:
Do we need this? I wonder if we use the IP somewhere, if not everything can just become a `Hostname`

Reply:
No ValueObjects\Web\IPAddress is not used in udb3-silex. I'll remove it.